proposals/low-level-struct-improvements.md

```diff
@@ -1569,6 +1605,42 @@ readonly ref struct SpanOfOne
 
 This means we must choose the shallow interpretation of `readonly`.
 
+#### Modeling constructors 
+One subtle tricky design point is how are constructors bodies modeled for ref safety? Essentially how is the following constructor analyzed? 
+
+```c#
+ref struct S
+{
+    int field;
+
+    public S(ref int f)
+    {
+        field = ref f;
+    }
+}
+```
+
+There are roughly two approaches:
+
+1. Model as a `static` method where `this` is a local where it's *safe-to-escape* is *calling method*
+2. Model as a `static` method where `this` is a `ref` or `out` parameter. 
```

Comment:
>where `this` is a `ref` or `out` parameter [](http://example.com/codeflow?start=30&length=42)

Could we drop the `ref` case and use "... where `this` is an `out` parameter" instead?